testing-plugin/src/main/kotlin/arrow/meta/plugin/testing/TestingLibraryWrapper.kt

```diff
@@ -10,14 +10,14 @@ import java.nio.file.Paths
 
 private const val DEFAULT_FILENAME = "Example.kt"
 
-internal data class CompilationResult(
+data class CompilationResult(
   val actualStatus: CompilationStatus,
   val log: String,
   val actualGeneratedFilePath: Path,
-  val classesDirectory: File
```

Comment:
There are other directories and this is only the classes directory. For instance:

```Paths.get(internalResult.outputDirectory.parent, "sources", "$DEFAULT_FILENAME.meta")```

is jumping to the parent in order to access to sources. I think if we talk about classes directory, the source code communicates better.